examples/g_spiral1.lisp

```diff
@@ -0,0 +1,27 @@
+; animated recusive spiral
+; click on "toggle animation"
```

Comment:
Instead, just start the animator `(def start (get ronin "animate"))`, and `(start)`